File: codes/python/python/prime.py
n=int(input("enter a number\n"))
count=0
i=1
while n>=i:
    if n%i==0:
        count=count+1
    i=i+1
if count==2:
    print("the number",n, "is prime")
else:
    print("the number",n, "is not prime")           

File: codes/c tutorial/prime.c
#include<stdio.h>
int main()
{
    int i,n,count=0;
    printf("Enter a number\n");
    scanf("%d",&n);
    for(i=1; i<=n; i++)
    {
if(n%i==0)
{
    count++;
}
    }
if(count==2)
{
    printf("number is prime");
}
else
{
    printf("number is not prime");
}
    return 0;

}